src/policy.rs

```diff
@@ -30,36 +34,42 @@ impl PolicyAxis {
         }
     }
 
+    /// Returns the number of attributes belonging to this axis.
     #[allow(clippy::len_without_is_empty)]
     #[must_use]
     pub fn len(&self) -> usize {
         self.attributes.len()
     }
 
+    /// Returns the list of attribute names belonging to this axis.
     pub fn attributes(&self) -> &[String] {
         &self.attributes
     }
 
+    /// Returns the axis name.
     pub fn name(&self) -> &str {
         &self.name
     }
 
+    /// Return `true` if this axis is hierarchical.
     pub fn is_hierarchical(&self) -> bool {
         self.hierarchical
     }
 }
 
-/// A policy is a set of fixed policy axes, defining an inner attribute
-/// element for each policy axis attribute a fixed number of revocation
-/// addition of attributes is allowed
+/// A policy is a set of policy axes. A fixed number of attribute creations
+/// (revocations + additions) is allowed.
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct Policy {
-    pub last_attribute_value: u32,
-    pub max_attribute_value: u32,
-    /// store the policies by name
-    pub store: HashMap<String, (Vec<String>, bool)>,
+    /// Last value taken by the attriute.
+    last_attribute_value: u32,
+    /// Maximum attribute value. Defines a maximum number of attribute
+    /// creations (revocations + addition).
+    max_attribute_value: u32,
```

Comment:
shouldn't this be `max_attribute_creations`?

Reply:
It can, indeed.